src/Jackett.Common/Content/index.html

```diff
@@ -215,7 +215,16 @@ <h3>Jackett Configuration</h3>
         </div>
         <div class="input-area">
             <span class="input-header">Cache enabled (recommended): </span>
```

Comment:
Very minor suggestion, this should probably be changed to something like `Cache method`. That doesn't sound quite right either, but hopefully you get the point.